text-input.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Text Input
+sidebar: home_sidebar
+permalink: /text-input/
+---
+
+* TOC Placeholder
+{:toc}
+
+## Introduction
+
+This page describe how to setup basic text input for Flutter applications.
```

Comment:
Seems like it should be "set up":
https://www.visualthesaurus.com/cm/wc/is-it-set-up-set-up-or-setup/

Fixed.